api/classes/errors.js

```diff
@@ -1,43 +1,70 @@
-var Boom = require('boom');
+"use strict";
 
-module.exports.attrs =  function(request, reply, source, error) {
-  var failedAttribute = '`' + error.data.details[0].path + '`';
+const Boom = require(`boom`);
 
-  // Look to see if the attribute was passed at all
-  if (error.data.details[0].message.indexOf('is required') !== -1) {
-    return reply(Boom.badRequest(failedAttribute + ' must be passed.', {
-      debug: true,
-      error: failedAttribute + ' must be passed.'
-    }));
+class Errors {
+  static attrs(request, reply, source, error) {
+    const errorDetails = error.data.details[0];
+    const failedAttribute = `\`${errorDetails.path}\``;
```

Comment:
Well I think this was meant for delimiting the path from the rest of the text in a log. It seems to be a convention followed throughout the code.